gh-weblog/components/Editor.jsx

```diff
@@ -12,25 +13,30 @@ export default createClass({
   render() {
     return (
       <textarea
-        ref="textarea"
+        ref={this.refs.textarea}
```

Comment:
this is... weird: if refs is an array then `this.refs[0]`, sure, but `this.refs.textarea` means `refs` should have been an object.

Reply:
I defined `refs` as an array of ref names in the `createClass` wrapper, but it could also be made explicit (you would pass refs in like `{textarea: createRef()}`). I don't think we ever need to pass an initial value into `createRef` though, which is why I went with the array of names.